lighthouse-core/fraggle-rock/user-flow.js

```diff
@@ -176,6 +166,69 @@ class UserFlow {
     const flowResult = await this.createFlowResult();
     return generateFlowReportHtml(flowResult);
   }
+
+  /**
+   * @return {LH.UserFlow.FlowArtifacts}
+   */
+  createArtifactsJson() {
+    /** @type {LH.UserFlow.GatherStep[]} */
+    const stepArtifacts = this._gatherSteps.map(step => ({
+      name: step.name,
+      artifacts: step.artifacts,
+      config: step.config,
+      configContext: step.configContext,
+    }));
```

Comment:
why map this? could just pass the array directly, or at most do a shorthand shallow clone with `[...this._gatherSteps]`

Reply:
`ActiveGatherStep` contains `runnerOptions` which are not serializable. This was to remove that param.